modules/core/include/opencv2/core/utility.hpp

```diff
@@ -56,6 +56,10 @@
 #include "opencv2/core.hpp"
 #include <ostream>
 
+#if __cplusplus >= 201103L
```

Comment:
MSVC defines `__cplusplus` to 199711
Probably we should target for MSVS 2013+ too.
Check [this condition](https://github.com/opencv/opencv/blob/master/modules/core/include/opencv2/core/cvdef.h#L366).